index.html

```diff
@@ -0,0 +1,240 @@
+    />
+  </head>
+  <body>
+    <section class="main">
+      <div class="topbar">
+        <a class="text-decoration-none me-4" href="">
+          <i class="fa-brands fa-facebook"></i>
+        </a>
+        <a class="text-decoration-none me-4" href="">
+          <i class="fa-brands fa-twitter"></i>
+        </a>
+        <a class="text-decoration-none me-4" href="">Log Out</a>
+        <a class="text-decoration-none me-4" href="">My Page</a>
+        <a class="text-decoration-none me-4" href="">Burmese</a>
+      </div>
+      <section class="wrap pb-5">
+        <header class="menu-wrap">
+          <nav>
+            <input
+              type="checkbox"
+              class="toggler"
+              id="toggle"
+              onclick="toggle()"
+            />
+            <div class="hamburger"><div></div></div>
+            <div class="menu">
+              <div class="overlay">
+                <div>
+                  <ul class="menubar">
+                    <li>
+                      <a href="#" class="brand">
+                        <img src="./img/logo.png" alt="logo" />
+                      </a>
+                    </li>
+                    <li>
+                      <ul class="nav-wrap">
+                        <li>
+                          <a href="about.html" class="nav-links about-link">
+                            About
+                          </a>
+                        </li>
+                        <li><a href="#" class="nav-links">Program</a></li>
+                        <li>
+                          <a href="#works" class="nav-links">Buy Tickets</a>
+                        </li>
+                        <li><a href="#works" class="nav-links">Sponsor</a></li>
+                        <li><a href="#works" class="nav-links">News</a></li>
+                        <li class="cc">
+                          <a href="index.html" class="textcc">
+                            Fundraising Concert
+                          </a>
+                        </li>
+                      </ul>
+                    </li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+          </nav>
+        </header>
+        <div class="intro">
+          <p id="introText">"Hello! Sharing World"</p>
+          <h1 class="heading mb-4">
+            Fund Raising Concert
+            <br />
+            for Displaced People
+            <br />
+            in Myanmar 2023
+          </h1>
+        </div>
+        <div class="mb-sm-5 detail-wrap">
+          <p class="detail p-2">
+            A fund raising concert for all the Displaced people in Myanmar after
+            the coup, creating a positive change with people from all over the
+            country and around the world is taking place in April, in Myanmar
+          </p>
+          <h2 class="day">2023.04.07(FRI) ~ 10(SUN)</h2>
+          <p class="place">@National Kandawgyi Gardens</p>
+        </div>
```

Comment:
- [ ] This section of the page should be wrapped in a `section` tag for semantic purposes and not a `div` tag. Kindly fix this

- [ ] Kindly align the paragraph to have equal margins to the left and right of the mobile view as is in the given design

Your design
![image](https://user-images.githubusercontent.com/81932478/219500174-9cc4ab28-7edb-4b0c-8fb6-177dd49eef49.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/219499987-07f47b9d-0f21-4b27-8946-7cad6d50a087.png)